pkg/apis/networking/v1alpha1/ingress_types.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha1
+
+import (
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:shortName=mcing
```

Comment:
@GitHubxsy Are you ok with the short name `mcing`?
How do you think `mci`? 

By the way, the short name of Ingress in Kubernetes is `ing`.

Reply:
The `mci` would be more popular.

Reply:
update it to `mci`.